packages/svelte-ui/components/Button.svelte

```diff
@@ -32,15 +27,15 @@
   class={isOutline
     ? `btn btn-outline btn-${size} gap-2`
     : `btn btn-${bg} btn-${size} gap-2`}
-  on:click|preventDefault={handleClick}
+  on:click={handleClick}
 >
   {#if isProcessing}
     <span class="loading loading-spinner animate-spin" />
   {:else}
     {#if icon}
```

Comment:
I think because we are showing different icons based on the request state, its better to keep it in the button component instead of lifting it outside too as a named slot